test/DebugInfo/basic.swift

```diff
@@ -36,8 +42,10 @@ public
 func foo(_ a: Int64, _ b: Int64) -> Int64 {
      var a = a
      var b = b
-     // CHECK-DAG: !DILexicalBlock(scope: ![[FOO]],{{.*}} line: [[@LINE-3]], column: 43)
-     // CHECK-DAG: ![[ASCOPE:.*]] = !DILocation(line: [[@LINE-4]], column: 10, scope: ![[FOO]])
+     // DWARF-CHECK-DAG: !DILexicalBlock(scope: ![[FOO]],{{.*}} line: [[@LINE-3]], column: 43)
+     // CV-CHECK-DAG: !DILexicalBlock(scope: ![[FOO]],{{.*}} line: [[@LINE-4]])
+     // DWARF-CHECK-DAG: ![[ASCOPE:.*]] = !DILocation(line: [[@LINE-5]], column: 10, scope: ![[FOO]])
+     // CV-CHECK-DAG: ![[ASCOPE:.*]] = !DILocation(line: [[@LINE-6]], scope: ![[FOO]])
```

Comment:
It looks like all the CV-CHECK-DAG are equivalent to the DWARF-CHECK-DAG checks. What about just separating out the DwarfVersion check into its own prefix?

Reply:
They are the same except CV-CHECK-DAG is missing the columns. What do you mean separating out the check into its own prefix? Didn't I already do that?

Reply:
From a maintenance perspective, I think it would be better to split out testing of the column generation into a separate columns test case and just have one set of CHECK-DAGs here. Otherwise this test gets really hard to read and update.

Reply:
Why would it be hard? Couldn't you just move the columns into a separate columns.swift test and then use the same CHECKs for codeview and dwarf?

Reply:
For example, in line 48 we need to get the value of ``ASCOPE``, but since CodeView doesn't have columns it won't be able to find that ``DILocation()``.  I suppose that I could use ``CHECK-SAME`` to fix that, but I wonder if that would be more messy.

Reply:
Sorry, what I meant was to remove all mentions of columns from basic.swift and create a new test file columns.swift that only tests columns. Would that work for you?

Reply:
Oh ok, that makes perfect sense.